securedrop/tests/client_test.py

```diff
@@ -65,19 +65,19 @@ def test_aad_initial_registration_succeeds(self):
 
     def assert_initial_registered_users_dict_is_valid(self, d):
         for email, cd in d.items():
-            self.assertEqual(email, "email_v")
+            self.assertEqual(email, "05c0f2ea8e3967a16d55bc8894d3787a69d3821d327f687863e6492cb74654c3")
             self.assertEqual(cd["email"], email)
-            self.assertEqual(cd["name"], "name_v")
-            self.assertTrue(not cd["contacts"])
+            self.assertTrue(cd["name"], ["data"])
```

Comment:
What is this supposed to be doing? Is the comma not supposed to be there?

Reply:
Yep, comma is not supposed to be there. 
It's supposed to make sure that cd[name][data] and cd[name[verify] both exist in the data written to disc.